environment.yml

```diff
@@ -0,0 +1,221 @@
+name: geo_deep_env
```

Comment:
love the idea

Reply:
yes! am aware Yves C made those changes. I would merge.